packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -27,15 +28,16 @@ const isTrueExactMatch = (
   houseNumberFull &&
   stripString(match?.houseNumberFull) === stripString(houseNumberFull);
 
-const LocationFinder: React.FC<{
-  focus: boolean;
-  matomoTrackEvent: Function;
-  sessionAddress: any; // @TODO replace any with address type.
-  setAddress: Function;
-  setErrorMessage: Function;
-  setFocus: Function;
-  topic: any; //@TODO: Replace it with IMTR-Client's TopicType
-}> = ({
+const LocationFinder: React.FC<
+  {
+    focus: boolean;
+    sessionAddress: any; // @TODO replace any with address type.
+    setAddress: (arg0: boolean) => void;
+    setErrorMessage: (arg0: any) => void;
```

Comment:
Would be very nice if we have an Error type here instead of any.

Reply:
Good thought, fixed it